package/CHANGELOG

```diff
@@ -21,6 +21,9 @@ The rules for this file:
  * 2.6.0
 
 Fixes
+  * Fix Boltzmann constant API break by enabling access to old
```

Comment:
In deprecations please

Reply:
Okay, I'll make that change now

Reply:
Also the entry really should be "incorrectly spelt Boltzman unit entry will be removed in version 2.x.x"